src/components/dashboard/SendTokens/Dash/FormContainer.jsx

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
Is it possible to merge Dash/FormContainer.jsx and Dash/Form.jsx somehow?

I'm asking, because connect is using in Form.jsx and Form.jsx is actual container. And FormContainer doesn't connected to store..

Reply:
That's true. Not just dash transfer form but every transfer form consists of two containers (the `*Form.jsx` and `*FormContainer.jsx`). We can easily unite `Dash/Form` with `Dash/FormContainer` but quite the contrary I would like to suggest to split the the transfer forms into a set of files. There is a lot of similar functionality among the transfer forms. I would like to extract reusable components and classes.